sdk/identity/Azure.Identity/src/AzureCliCredentialClient.cs

```diff
@@ -31,6 +31,8 @@ internal class AzureCliCredentialClient
         // The default install paths are used to find Azure CLI if no path is specified. This is to prevent executing out of the current working directory.
         private static readonly string DefaultPathWindows = $"{EnvironmentVariables.ProgramFilesX86}\\Microsoft SDKs\\Azure\\CLI2\\wbin;{EnvironmentVariables.ProgramFiles}\\Microsoft SDKs\\Azure\\CLI2\\wbin";
         private const string DefaultPath = "/usr/bin:/usr/local/bin";
+        private const string DefaultWorkingDir = "/bin/";
+        private static readonly string DefaultWorkingDirWindows = Environment.GetFolderPath(Environment.SpecialFolder.System);
```

Comment:
Can this variable be null ?
If so, should we should do the null check, if the process execution defaults to user directory when null dir path is provided.

Reply:
This is guaranteed to be non-null on windows which is one of the reasons I opted for this over just reading the `SYSTEMROOT` environment variable. This does make the behavior slightly different then other langauges where reading `SystemRoot` is the best solution available, but I think it varies towards the behavior we would want rather than the other direction.

Reply:
on Windows 95, 98 and ME, this variable will be null.
But, hopefully no one is using them now.

Reply:
The KNOWNFOLDER IDs (previous CSIDL GUIDs were used) are what defines the default environment variables. Besides, Win9x is LONG out of support and doesn't support any .NET platforms we support. 😉